src/main/java/com/neonlab/product/apis/CreateOrderApi.java

```diff
@@ -0,0 +1,41 @@
+package com.neonlab.product.apis;
+import com.neonlab.common.dto.ApiOutput;
+import com.neonlab.common.expectations.InvalidInputException;
+import com.neonlab.common.utilities.StringUtil;
+import com.neonlab.product.dtos.OrderDto;
+import com.neonlab.product.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import java.util.Objects;
+
+
+@Service
+public class CreateOrderApi {
+
+    @Autowired
+    private OrderService orderService;
+
+
+    public ApiOutput<?> createOrder(OrderDto orderDto) {
+        try {
+            Validate(orderDto);
+           return orderService.createOrder(orderDto);
+
+        } catch (InvalidInputException e) {
+            return new ApiOutput<>(HttpStatus.BAD_REQUEST.value(), e.getMessage());
+        }
+    }
+
+    public void Validate(OrderDto orderDto) throws InvalidInputException {
+        if(StringUtil.isNullOrEmpty(orderDto.getPaymentId())){
```

Comment:
I asked you to learn `@NotNull` and `@NotBlank` so that you can use it here. Please learn the same and refactor the code accordingly.

cc- @sjha24